.github/workflows/e2e-test.yml

```diff
@@ -119,18 +123,23 @@ jobs:
         shell: bash
         run: |
           node e2e/runner/run_cypress_ci.js test \
-            --env grepTags="-@mongo+-@flaky+-@OSS --@quarantine",grepOmitFiltered=true \
+            --env grepTags="-@mongo+-@flaky+-@visual+-@OSS --@quarantine",grepOmitFiltered=true \
```

Comment:
Same for a tag name. Right now it is the `@visual`. Can adjust to something like `@visual-e2e`